CHANGELOG.md

```diff
@@ -5,7 +5,9 @@
 * The default logging format has been changed. See [#9055](https://github.com/influxdata/influxdb/pull/9055) for details.
 
 ### Features
-
+- [#9146](https://github.com/influxdata/influxdb/issues/9146): Backup/Restore can produce/consume data in the same format as the enterprise backup/restore tool. 
+- [#8880](https://github.com/influxdata/influxdb/issues/8879): Restore runs in online mode
+- [#8879](https://github.com/influxdata/influxdb/issues/8879): Export functionality using start/end to filter exported data by timestamp
```

Comment:
Was it necessary that all of these changes be included in a single pull request? If not, I would prefer to see each feature have a separate PR. Combining them makes it more difficult to review and know which changes are in support of which feature.